### ISSUE TITLE
Cannot send messages/commands

### ISSUE THREAD

**Issue by ploz** [JFreegman/toxic#390]
Fairly certain this started happening after [multiline input support](https://github.com/JFreegman/toxic/commit/e6f839f9ac635b5b6bb47583c666e5c700cb5c9b) was added. Neither "Enter" or "Ctrl+J" can send a message/command, no matter what paste-mode or the current tab is set to. When paste-mode is off, "Enter" seems to do nothing and "Ctrl+J" sends a pilcrow (¶); while on both keys send pilcrows. Tested in Rxvt-Unicode, though problem persists in both termite and xterm.

**Comment by JFreegman** [comment]
This seems like a pretty serious bug. Can you give me more information about your system and your version of toxic?

**Comment by ploz** [comment]
##### SYSTEM INFORMATION

```
KERNEL: Linux 4.4.3-1-ARCH #1 SMP PREEMPT Fri Feb 26 15:09:29 CET 2016 x86_64 GNU/Linux
WM: bspwm-git 1128-1 (no toxic keys are being eaten by sxhkd or anything else)

Current broken version: toxic-git 0.7.0.r28.g5b29ce7-1 [0.7.0_r1686]
Last working version: toxic-git 0.7.0.r22.geb02424-1 [0.7.0_r1680]
Terminal emulator: rxvt-unicode 9.22-1

ncurses 6.0-4
curl 7.47.1-1
desktop-file-utils 0.22-1
freealut 1.1.0-5
libnotify 0.7.6-2
qrencode 3.4.4-1
```

No change when running toxic in virtual console. I've temporarily downgraded to r1680 so toxic is usable.

**Comment by JFreegman** [comment]
@mall0c any ideas?

**Comment by mphe** [comment]
Sounds strange.
Have you tried removing your .inputrc and .bashrc (or whatever shell you use)? Does the problem persist?

**Comment by ploz** [comment]
@mall0c no change

**Comment by mphe** [comment]
Please clone [this repo](https://github.com/mall0c/toxic/tree/keytest) and compile the "keytest" branch.
I added some lines to print the keycode when a key is pressed. Ctrl-J should output 10 and Return 13.
If that's not the case please post the keycodes you get.

**Comment by ploz** [comment]
@mall0c Keys seem to be outputting as intended. `Ctrl-J = 10`, `Return = 13`. Same output regardless of paste-mode setting.

**Comment by ploz** [comment]
I've figured out what the problem is. For some reason when a key is bound to `Ctrl+M` (or `Ctrl+m`) it breaks sending messages. This is the keys section of my config when it breaks toxic:

```
keys = {
  next_tab="Ctrl+P";
  prev_tab="Ctrl+O";
  scroll_line_up="PAGEUP";
  scroll_line_down="PAGEDOWN";
  half_page_up="Ctrl+B";
  half_page_down="Ctrl+F";
  page_bottom="Ctrl+M"; //Key that breaks toxic
  peer_list_up="Ctrl+[";
  peer_list_down="Ctrl+]";
  toggle_peerlist="Ctrl+V";
  toggle_paste_mode="Ctrl+T";
};
```

I tested `Ctrl+M/m` with a few other keys other than just `page_bottom` and they've all still broken toxic. I am unsure whether or not it is the full binding `Ctrl+M/m` which causes the break of just the `m` part.

**Comment by DerVerruckteFuchs** [comment]
I can confirm that binding anything to `Ctrl+m` breaks sending messages. I tested having `Ctrl+m` commented out and having not commented out. Bound: I can type a command, input it, then have nothing happen. Unbound: I can type a command and have it work normally.

I have my `Ctrl+m` bound to `half_page_down`, that's when it works of course.

It's an oddly specific bug, but at least there's a decent work around for now. Now I can finally test if audio has been fixed.
